app/models/audit_results.rb

```diff
@@ -1,17 +1,6 @@
 # frozen_string_literal: true
 
-# AuditResults allows the correct granularity of information to be reported in various contexts.
-#   By collecting all the result information in this class, we keep the audit check code cleaner, and
-#   enable an easy way to provide:
-#    - the correct HTTP response code for ReST calls routed via controller
-#    - error information reported to workflows
-#    - information to Rails log
-#    - in the future, this may also include reporting to the provenance database as well
-#
-# All results are kept in the result_array attribute, which is returned by the report_results method.
-#   result_array = [result1, result2]
-#   result1 = {response_code => msg}
-#   result2 = {response_code => msg}
```

Comment:
I'm not in love with these comments, but they are informative.  What is the thinking behind removing them ... and not adding this info anywhere else (e.g. in the "beginners guide" that doesn't exists yet ... but there is a ticket that could get this info pasted as a comment)

Reply:
Good call.

Reply:
I put some of this back in.